src/modules/rules/rules.service.ts

```
@@ -0,0 +1,58 @@
+import { CreateRuleDto } from './dto/create-rule.dto';
+import { UpdateRuleDto } from './dto/update-rule.dto';
+
+@Injectable()
+export class RulesService {
+  constructor(
+    @InjectRepository(Rule)
+    private readonly ruleRepository: Repository<Rule>,
+  ) {}
+
+  private validateThresholds(dto: CreateRuleDto | UpdateRuleDto) {
+    if (dto.similarityThreshold !== undefined && dto.deviationAllowedPct !== undefined) {
```

ESLint

[error] Replace `dto.similarityThreshold·!==·undefined·&&·dto.deviationAllowedPct·!==·undefined` with `⏎······dto.similarityThreshold·!==·undefined·&&⏎······dto.deviationAllowedPct·!==·undefined⏎····`

(prettier/prettier)